src/Polytopes/iterators.jl

```diff
@@ -97,9 +97,26 @@ Halfspace(a::AbstractVector, b) = Halfspace(reshape(a, 1, :), b)
 
 Halfspace(a) = Halfspace(a, 0)
 
+@doc Markdown.doc"""
+    Hyperplane(a, b)
+
+One hyperplane `H(a,b)` is given by a vector `a` and a value `b` such that
+$$H(a,b) = \{ x | ax = b \}.$$
+"""
+struct Hyperplane
+    a::Polymake.Matrix{Polymake.Rational}
```

Comment:
```suggestion
    a::Polymake.Vector{Polymake.Rational}
```
Is there a reason why this is a `Polymake.Matrix` instead of a `Polymake.Vector`; apart from matching what we already use in `Halfspace` ?